gobblin-temporal/src/main/java/org/apache/gobblin/temporal/workflows/metrics/EventSubmitterContext.java

```diff
@@ -40,7 +47,7 @@
  */
 @Getter
 public class EventSubmitterContext {
-  private final List<Tag<?>> tags;
+  private List<Tag<?>> tags;
```

Comment:
`final` could remain if we make this immutable.

IMO, every field being `final` is the sign of a healthy abstraction: fully thread-safe and pre-disposed against mistaken/careless usage patterns